docs/common_issues.md

```diff
@@ -34,4 +34,12 @@ BINARY_CONTENT_TYPES=image/*,font/*
 
 will let the micro serve these files. 
 
-Please, use the [`deta update -e [env_file_name]`](https://docs.deta.sh/docs/cli/commands#deta-update) command to update the environment variables of the micro.
\ No newline at end of file
+Please, use the [`deta update -e [env_file_name]`](https://docs.deta.sh/docs/cli/commands#deta-update) command to update the environment variables of the micro.
```

Comment:
Please, use the [`deta update -e [env_file_name]`](https://docs.deta.sh/docs/cli/commands#deta-update) command to update the environment variables of the Micro.